frontend/public/components/graphs/_graphs.scss

```diff
@@ -78,8 +78,17 @@
 }
 
 .query-browser__table-toggle {
+  cursor: pointer;
```

Comment:
This is likely an accessibility problem we should fix.

Reply:
👍 Good point, thanks